fpga/Makefile

```diff
@@ -73,6 +73,22 @@ default: $(mcs)
 fpga_dir := $(base_dir)/fpga/fpga-shells/$(FPGA_BRAND)
 fpga_common_script_dir := $(fpga_dir)/common/tcl
 
+#########################################################################################
+# setup misc. sim files
+#########################################################################################
+SIM_FILE_REQS += \
+	$(ROCKETCHIP_RSRCS_DIR)/vsrc/EICG_wrapper.v
+
+# copy files but ignore *.h files in *.f (match vcs)
+$(sim_files): $(SIM_FILE_REQS) | $(build_dir)
+	mkdir -p $(dir $(sim_files))
```

Comment:
Since you have `| $(build_dir)` in the deps for the rule, I don't think you need this `mkdir` anymore.